packages/theming/theme-tokens/src/pipeline-input/token-input-global.macos.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://raw.githubusercontent.com/microsoft/fluentui-token-pipeline/main/docs/token.schema.json",
+  "Meta": {
+    "FluentUITokensVersion": 0
+  },
+  "Global": {
+    "Font": {
+      "Family": {
+        "Default": { "value": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"" },
+        "Monospace": { "value": "\"Courier New\"" },
+        "Numeric": { "value": "\"Helvetica Neue\", Helvetica, Arial, \"Microsoft Sans Serif\"" }
+      }
+    }
```

Comment:
Shouldn't this be 'System', AKA Apple's San Fransisco font?

Reply:
Hmm I'm not sure, will need to revisit the font tokens... let me create a bug